docs/developers/releasing.md

```diff
@@ -0,0 +1,106 @@
+# Releasing
+Karmada has two types of releases, major release, and minor release. For example, `v1.3.0` is a major release, and `v1.3.1` is a minor release. The relationship between release, tag, and branch is as follows:
+![img](../resources/developers/releasing.png)
+
+For different release types, the procedure is different.
+
+## Major release
+The major version should be released from master branch. The release procedure is described as follows.
```

Comment:
Even though we are tagging the `minor` release from the master branch for now, but I think we should tag it from the release branch.

Reply:
fixed